pkg/skaffold/build/kaniko/args.go

```diff
@@ -188,6 +188,9 @@ func Args(artifact *latest.KanikoArtifact, tag, context string) ([]string, error
 
 	args = append(args, labels...)
 
+	// setting the flag --digest-file=/dev/termination-log will write the digest to that file, which is picked up by Kubernetes automatically as the {{.state.terminated.message}} of the container.
+	args = append(args, "--digest-file", "/dev/termination-log")
```

Comment:
`/dev/termination-log` can be the default value of `--digest-file`. When `--digest-file` is set, pass it to  `v1.Container.TerminationMessagePath` in `#kanikoPodSpec`, since [digestFile](https://skaffold.dev/docs/references/yaml/#build-artifacts-kaniko-digestFile) is exposed in skaffold API.

Reply:
fixed. Thanks!